src/Hug.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Card from './Card'
+import Flex from './Flex'
+import Icon from './Icon'
+import Text from './Text'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+const StyledCard = styled(Card)`
```

Comment:
NB: didn't we discuss using a default border-radius on the Card component itself? cc @odmln

Reply:
@jxnblk yes, a 2px border radius